src/App.vue

```diff
@@ -18,11 +20,13 @@ export default {
   name: "MainComponents",
   data() {
     return {
-      username: localStorage.getItem("token"),
+      username: localStorage.getItem("token").toUpperCase(),
```

Comment:
@JatinThakur14 If the token isnt there in localstorage, this will throw an error.

Reply:
It will always be in localStorage, because we can't access the map unless we are logged in. and If we are logged in it will have a token. but I will change it just in case